ansible/roles/config_containerd/templates/config.toml.tmpl

```diff
@@ -62,7 +62,7 @@ imports = ["/etc/containerd/conf.d/*.toml"]
     stream_server_port = "0"
     stream_idle_timeout = "4h0m0s"
     enable_selinux = false
-    sandbox_image = "{{ default_image_registry }}{{ k8s_image_registry }}/pause:{{ pause_image_version }}"
+    sandbox_image = "{{ pause_image | default('k8s.gcr.io/pause:3.6') }}"
```

Comment:
This is just a fall back to get `containerd` to start when `download_images` is `true` and `pause_image` is not set. The `images` role will set the correct `pause_image` and then reconfigure `containerd`.

Reply:
This has been removed and we will always have a pause_image set by the time `config` is run.